lib/styles/react-joyride.scss

```diff
@@ -221,7 +221,7 @@ $joyride-tooltip-arrow-scale: ($joyride-tooltip-arrow-size / ($joyride-tooltip-a
     &.right {
       margin-left: $joyride-tooltip-arrow-height;
 
-      .triangle {
+      .joyride-tooltip__triangle {
```

Comment:
Selector `joyride-tooltip__triangle` should be written in lowercase with hyphens